awkward-cpp/include/awkward/common.h

```diff
@@ -58,7 +58,6 @@ extern "C" {
     const char* filename;
     int64_t identity;
     int64_t attempt;
-    bool pass_through;
   };
```

Comment:
I spoke incorrectly: you are actually removing the `pass_through` field from the `Error` struct.

I looked through all the files in https://github.com/scikit-hep/awkward/tree/main/src/awkward/_connect/numba and I don't see anything that references the `Error` struct, so I guess all of that has been removed. (Earlier, some Numba code was calling these kernels directly; that was part of the design decision to make them purely `extern "C"`. Calling external function pointers from Numba is slow—slower than would be expected of just not being able to optimize around it, for some reason—and so we gave that up. There are apparently no further traces of it. But if there were any, if Numba called kernels and expected this `Error` struct in return, it would have to make an assumption about its memory layout, and we wouldn't want those assumptions to get out of sync.)

Short story: it looks to me like it's okay to change the `Error` struct, so full speed ahead.